packages/common/src/utils.js

```diff
@@ -457,3 +458,20 @@ export function defineAlias(src, target, prop, opts = {}) {
     }
   })
 }
+
+export function serializeFunction(func) {
+  let open = false
+  return serialize(func)
+    .replace(/^(\s*):(\w+)\(/gm, (_, spaces) => {
+      return `${spaces}:function(`
+    })
+    .replace(/^(\s*)(\w+)\s*\((.*?)\)\s*\{/gm, (_, spaces, name, args) => {
+      if (open) {
+        return `${spaces}${name}:function(${args}) {`
+      } else {
+        open = true
+        return _
```

Comment:
General tip: _ as function argument is usually for ignoring that arg. (IDEs and Eslint can imply this and don't warn about unused arg)

Reply:
@pi0 yeah, I initially used it with this purpose, but the algorithm changed -- since it's meant to be discarded after the 1st iteration I thought it was OK to leave as-is.